src/inflight_activation_store.rs

```diff
@@ -73,6 +73,9 @@ pub struct InflightActivation {
     /// depending on the retry policy of an activation it will either be deadlettered or discarded.
     pub remove_at: DateTime<Utc>,
 
+    // If the task has specified an expiry, this is the timestamp after which the task should be removed from inflight store
```

Comment:
```suggestion
    /// If the task has specified an expiry, this is the timestamp after which the task should be removed from inflight store
```

The extra / makes it a docstring that shows up in LSP.